homeassistant/components/mobile_app/webhook.py

```diff
@@ -96,6 +97,9 @@
 
     data = webhook_payload
 
+    _LOGGER.debug("Received webhook payload for type %s: %s", webhook_type,
+                  json.dumps(data))
```

Comment:
Is this a hot code path?

If so, this will execute json.dumps even when debug logging is disabled. We might want to just use the defaulr __str__ representation instead.

Reply:
Fixed